src/util-daemon.c

```diff
@@ -110,7 +110,14 @@ void Daemonize (void)
               through conf file */
 
     /* Creates a new process */
+#if defined(OS_DARWIN) && defined(__clang__)
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+#endif
     pid = fork();
```

Comment:
Is `fork()` deprecated on MacOS?

Reply:
Looks like it in favor of spawn_posix ...

Reply:
Interesting:
```
The posix_spawn() and posix_spawnp() functions are used to create
       a new child process that executes a specified file.  These
       functions were specified by POSIX to provide a standardized
       method of creating new processes on machines that lack the
       capability to support the [fork(2)](https://man7.org/linux/man-pages/man2/fork.2.html) system call.  These machines
       are generally small, embedded systems lacking MMU support.
```